src/routes/postbox.ts

```diff
@@ -96,12 +94,11 @@ export const listPostboxItems = async (req, res) => {
   res.status(200).send(person.postboxItems || []);
 };
 
-const getPostboxItemById = async (postboxItemId: string) => {
-  const persons = await getAllPersons();
-  const allItems = [].concat(
-    ...persons.map(({ postboxItems }) => postboxItems || [])
+export const getPostboxItemById = async (postboxItemId: string) => {
+  const person = await findPerson(
+    (p) => !!(p.postboxItems ?? []).find((pb) => pb.id === postboxItemId)
```

Comment:
:thinking: it would be more readable to replace all `find` by `some` in `findPerson` fn.

Reply:
Related commit: https://github.com/kontist/mock-solaris/pull/114/commits/bc4e4071f154373cfe2a99b139caf81b9af58520